joomla/joomla-cms
# [4.0] Javascript clean up

Pull Request for Issue # .

### Summary of Changes
- remove or move code out of core.js
- Rename global functions to Joomla.functionName (for testing and because it's not nice to pollute to have global functions)
- fix the multi select console error


### Testing Instructions
Apply patch and test any list view (e.g. articles that ALL the links, buttons work as expected)
Try the sendmail in Global Configuration
And also in the global configuration [System Settings] try the button refresh for the Help Server


### Expected result



### Actual result



### Documentation Changes Required
